client/reader/user-profile/controller.tsx

```diff
@@ -34,6 +38,8 @@ export function userPosts( ctx: Context, next: () => void ): void {
 				analyticsPageTitle,
 				mcKey
 			) }
+			showBack={ !! ctx.lastRoute }
+			handleBack={ goBack }
```

Comment:
Can we use getPreviousRoute selector inside the component instead of passing these props?